jobsdb/backup.go

```diff
@@ -554,15 +561,21 @@ func (jd *Handle) createTableDumps(ctx context.Context, queryFunc func(int64) st
 			preferences, err := jd.conf.backup.fileUploaderProvider.GetStoragePreferences(workspaceID)
 			if errors.Is(err, fileuploader.NoStorageForWorkspaceError) {
 				offset++
-				jd.logger.Infof("getting storage preferences failed with error: %w for workspaceID: %s", err, workspaceID)
+				jd.logger.Debugw(
+					"getting storage preferences failed",
+					"workspaceID", workspaceID,
```

Comment:
where is the error?

Reply:
The only error possible here is workspace not being present in the server's config.
So didn't think it was important here.

Reply:
What will happen if someone changes the underlying implementation and a new error comes up? how this part of the code can be secured?
Previously I proposed to use a sentinel error for that exact purpose.